### PR TITLE
Feature/sort by aisle

### PR DESCRIPTION
**Pull request by mischb**
sorting and reverse sorting working. Two small bugs : 1) when adding new item they are added to the end.  2) items without aisle numbers don't get sorted (this might be fine)

to sort click the 'aisle' header  (ALSO. could probably refactor the utils but seeme dlike more work than it was worth)

### CODE REVIEW

**Comment by aseehra** [comment]
Personally, I think that adding items that don't have aisle numbers is fine.

File: src/reducers/utils.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export function compareAisle(a, b) {
```
Comment (aseehra) [inline src/reducers/utils.js]: Unfortunately, your compare aisle doesn't sort numerical items properly. Aisles are stored/sent as strings (as you can have aisles like "dairy" or "produce"). Right now, you are doing string comparisons---you end up with sorts like this:
localhost_3000_lists_5c40df593fe04f89a378b3fc

We also probably want the comparison function do something sane for things that aren't numbers.

**Comment by aseehra** [comment]
Per our conversation: if we want to keep the user's own entry order as the "canonical" version, we should switch to sorting on render, rather than storing the sorted version in redux. Instead keep the "sorted?" flag in redux

File: src/App.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Route, withRouter, Switch } from 'react-router-dom';
 import RegistrationPage from './components/auth-components/registration-page';
 import LoginPage from './components/auth-components/login-page';
 import Lists from './components/Lists';
-import Items from './components/Items';
+import Items from './components/Items/index';
```
Comment (aseehra) [inline src/App.js]: leave this as `import Items from './components/Items'`

Comment (mischb) [reply src/App.js]: for some reason i'm getting an error when i do that. any idea why that would ??

**Pull request merged**